src/components/StartQuizPage/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Button, Form, Input } from 'antd';
+
+import  './index.css';
+
+const QuizStart = () => {
+    return (
+        <div className="quizStart" >
+            <Form id='form'>
+                <Form.Item
+                    name="username"
+                    rules={[{
+                        required: true,
+                        message: 'Please enter your username!',
```

Comment:
If the username doesn't match the pattern, it still shows the same message, so we can change the message to "Please enter a valid username"

Reply:
yeah I'll update the changes